.github/PULL_REQUEST_TEMPLATE/new-language-code-snippet.md

```diff
@@ -0,0 +1,35 @@
+---
+name: New Language Code Snippet
+about: Add a code snippet in a new language
+title: Add {PROJECT} in {LANGUAGE}
+labels: enhancement, tests
+assignees: '@TheRenegadeCoder/core'
```

Comment:
I don't know that this will work. I currently can't assign core to a pull request, but I can request a review from core.

Reply:
You're right. Perhaps we try to assign it to reviewers instead.
It probably won't work for the issues either. :disappointed: